cmd/neofs-node/config.go

```diff
@@ -185,6 +187,13 @@ type cfgObject struct {
 	cfgLocalStorage cfgLocalStorage
 }
 
+type cfgNotifications struct {
+	enabled bool
+	nw      notificationWriter
+
```

Comment:
Could you please sync linebreaks between fields?

Reply:
sure